src/js/utils/responsive.js

```diff
@@ -29,3 +29,11 @@ export const getBreakpoint = (windowWidth, theme) => {
 
 export const getDeviceBreakpoint = (type, theme) =>
   theme.global.deviceBreakpoints[type];
+
+export const getBreakpointStyle = (theme, breakpointSize) => {
+  const breakpoint = breakpointSize && theme.global.breakpoints[breakpointSize];
```

Comment:
```suggestion
  const breakpoint = (breakpointSize && theme.global.breakpoints[breakpointSize]) || {};
```

Reply:
This will be a little more defensive in case `breakpointSize` isn't truth-y or there isn't a breakpoint in the theme at that size.

Reply:
Done